components/Search/Search.module.css

```diff
@@ -50,3 +50,50 @@
     display: none;
   }
 }
+
+.wrapper-autocomplete {
+  width: 100%;
+  height: 30px;
+  margin-right: var(--m-1);
+  margin-left: var(--m-1);
+
+  & :global(.aa-Form),
+  & :global(.aa-DetachedSearchButton) {
+    height: 30px;
+    border-radius: 24px;
+    border-color: var(--color-light-gray);
+  }
+  & :global(.aa-SubmitIcon) {
+    color: var(--color-lighter-gray);
+  }
+  & :global(.aa-Input) {
+    fontsize: var(--fs-text-md);
+  }
+
+  @media (max-width: 560px) {
+    width: 100%;
+  }
+}
+
+.aa-Form {
```

Comment:
Will this classname work without `:global`?

Reply:
No, it won't.